app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class StaticPagesController < ApplicationController
+  def home
+    if logged_in?
```

Comment:
You can move this `logged_in?` check to `before_action` callback in `ApplicationController`